
<launch>

    <arg name = "scholar_teleop_vx_speed" default = " 0.2 " />
	<arg name = "scholar_teleop_angular_speed" default = " 0.2 " />
	<arg name = "cmd_vel_new_name" default = " cmd_vel " />

	<node	pkg="scholar_teleop" 	type="scholar_teleop_bringup" 	name="scholar_teleop_bringup"  output="screen" required="true" cwd="node" >
        
        <param name = "scholar_teleop_vx_speed"    value = "$(arg scholar_teleop_vx_speed)" />
        <param name = "scholar_teleop_angular_speed"  value = "$(arg scholar_teleop_angular_speed)" />

        <param name = "cmd_vel_topic_name"  value = "$(arg cmd_vel_new_name)" />
        <remap from="/scholar_teleop_bringup/cmd_vel" to="cmd_vel"/>

	</node>
</launch>                                                                                         
